### PR TITLE
Change docker-compose sut spin up for node version testing.

### PR DESCRIPTION
**Pull request by fisehara**
Change-type: patch
Signed-off-by: fisehara <harald@balena.io>

### CODE REVIEW

**Comment by ab77** [comment]
LG2M @fisehara

File: package.json [hunk 1]
```diff
@@ -18,14 +18,9 @@
     "webpack-server": "grunt server",
     "webpack-build": "npm run webpack-browser && npm run webpack-module && npm run webpack-server",
     "lint": "balena-lint -e js -e ts src build typings Gruntfile.ts && npx tsc --project tsconfig.dev.json --noEmit",
-    "test": "npm run lint & npm run build && npm run webpack-build",
-    "test:compose": "npm run compose:build && npm run compose:postgres && docker-compose -f docker-compose.test.yml run sut npm run mocha",
-    "test:fast": "npm run compose:postgres && npm run mocha:local",
+    "test": "npm run lint & npm run build && npm run webpack-build && npm run test:dc",
+    "test:dc": "trap 'docker-compose -f docker-compose.npm-test.yml down ; echo Stopped ; exit 0' SIGINT; docker-compose -f docker-compose.npm-test.yml up -d && sleep 2 && DATABASE_URL=postgres://docker:docker@localhost:5431/postgres npm run mocha",
```
Comment (joshbwlng) [inline package.json:22]: Nit: I'd prefer `test:compose` to `test:dc` as its a bit clearer

Comment (fisehara) [reply package.json:22]: @joshbwlng works for me as well! Thanks.

**Comment by fisehara** [comment]
@balena-ci I self-certify!

**Pull request merged**